### PR TITLE
gsoc_idea project Dashboard Prototype

### PR DESCRIPTION
**Pull request by shaistha24**
Adding the first prototype images of the dashboard , the end result may change after coding. Python Dash Framework for dashboard creation will be used.

### CODE REVIEW

File: gsoc_dashboard/phase1_dashboard_prototype.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+### GSOC IDEA - Creation of Dashboard and integration of SAST tool to Auto-generate assessment scores!
```
Comment (rejahrehim) [inline gsoc_dashboard/phase1_dashboard_prototype.md]: Use Single #

File: gsoc_dashboard/phase1_dashboard_prototype.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+The whole project is divided into 3 phases *(Testing and documentation is also done in each phase)*
+1. Dashboard Creation 
+2. Integration of SAST tool 
+3. Auto-generation of score
+
+###### *This file follows the phase 1 i.e., Dashboard Creation*
```
Comment (rejahrehim) [inline gsoc_dashboard/phase1_dashboard_prototype.md]: Use ##

File: gsoc_dashboard/phase1_dashboard_prototype.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+### PHASE-1 : Dashboard Creation
```
Comment (rejahrehim) [inline gsoc_dashboard/phase1_dashboard_prototype.md]: Use exact header hierarchy.

File: gsoc_dashboard/phase1_dashboard_prototype.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+Tools:
+![](https://github.com/shaistha24/RiskAssessmentFramework/blob/master/gsoc_dashboard/pg6.jpg)
+Resources:
+![](https://github.com/shaistha24/RiskAssessmentFramework/blob/master/gsoc_dashboard/pg7.jpg)
+Settings:
+![](https://github.com/shaistha24/RiskAssessmentFramework/blob/master/gsoc_dashboard/pg8.jpg)
+FAQ:
+![](https://github.com/shaistha24/RiskAssessmentFramework/blob/master/gsoc_dashboard/pg9.jpg)
+
+###### *NOTE: Choice of language for coding is Python , framework: Dash*
```
Comment (rejahrehim) [inline gsoc_dashboard/phase1_dashboard_prototype.md]: Use exact header hierarchy.

File: gsoc_dashboard/phase1_dashboard_prototype.md (hunk 2 above)
Comment (shaistha24) [reply gsoc_dashboard/phase1_dashboard_prototype.md]: OK

File: gsoc_dashboard/phase1_dashboard_prototype.md (hunk 1 above)
Comment (shaistha24) [reply gsoc_dashboard/phase1_dashboard_prototype.md]: ok

File: gsoc_dashboard/phase1_dashboard_prototype.md (hunk 4 above)
Comment (shaistha24) [reply gsoc_dashboard/phase1_dashboard_prototype.md]: ok

File: gsoc_dashboard/phase1_dashboard_prototype.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+# Creation of Dashboard and integration of SAST tool for Automatic generation of assessment scores [GSOC IDEA]
```
Comment (rejahrehim) [inline gsoc_dashboard/phase1_dashboard_prototype.md]: Don't use "_" in the file name separate with "-".

Comment (shaistha24) [reply gsoc_dashboard/phase1_dashboard_prototype.md]: ok thank you

**Pull request merged**